packages/components/data-table-manager/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@commercetools-uikit/data-table-manager",
+  "version": "10.22.2",
+  "description": "",
+  "main": "dist/data-table-manager.cjs.js",
+  "module": "dist/data-table-manager.esm.js",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "prepare": "../../../scripts/version.js replace",
+    "prebuild": "rimraf dist",
+    "build": "cross-env NODE_ENV=production rollup -c ../../../rollup.config.js -i ./src/index.js"
+  },
+  "files": ["dist"],
+  "sideEffects": false,
+  "license": "MIT",
+  "dependencies": {
+    "@commercetools-uikit/accessible-button": "10.22.1",
+    "@commercetools-uikit/async-select-input": "10.22.0",
+    "@commercetools-uikit/collapsible-motion": "10.21.0",
+    "@commercetools-uikit/constraints": "10.21.0",
+    "@commercetools-uikit/design-system": "10.13.0",
+    "@commercetools-uikit/field-label": "10.22.1",
+    "@commercetools-uikit/grid": "10.18.4",
+    "@commercetools-uikit/hooks": "10.21.0",
+    "@commercetools-uikit/icons": "10.22.0",
+    "@commercetools-uikit/primary-button": "10.22.1",
+    "@commercetools-uikit/radio-input": "10.21.0",
+    "@commercetools-uikit/secondary-button": "10.22.1",
+    "@commercetools-uikit/secondary-icon-button": "10.22.1",
+    "@commercetools-uikit/select-input": "10.22.0",
+    "@commercetools-uikit/spacings": "10.18.5",
+    "@commercetools-uikit/text": "10.21.0",
+    "@commercetools-uikit/tag": "10.22.1",
+    "@commercetools-uikit/utils": "10.21.0",
+    "@emotion/core": "10.0.28",
+    "@emotion/styled": "10.0.27",
+    "debounce-promise": "^3.1.2",
+    "lodash": "4.17.15",
+    "lodash.differencewith": "4.5.0",
```

Comment:
Do we need this dependency since we already use `lodash`?

Reply:
We are only using that one. Maybe it's better to remove `lodash` instead...

Reply:
Yeah we should remove it from here and add `lodash/differencewith` to the list of `ignoredExternals` on `rollup.config.js` (for treeshaking purposes)